app/src/main/java/com/example/molliestephenson/tictactoe/MainActivity.java

```diff
@@ -48,13 +51,25 @@ private void createBoard() {
         mBoard.add(mRightBottom);
     }
 
-    private void createGame() {
-        Board board = new Board(3);
-        Player player1 = new MobilePlayer(Marks.X);
-        Player player2 = new MobilePlayer(Marks.O);
+    private void createGame(boolean isAHumanVHumanGame, boolean isA3x3Board) {
+        Board board = setBoardSize(isA3x3Board);
+        Player player1 = null;
+        Player player2 = null;
+        if (isAHumanVHumanGame) {
+            player1 = new MobilePlayer(Marks.X);
+            player2 = new MobilePlayer(Marks.O);
+        }
         this.mGameEngine = new GameEngine(player1, player2, board);
     }
 
+    private Board setBoardSize(boolean isA3x3Board) {
```

Comment:
Feels odd that this method takes a boolean, perhaps an Enum would be more flexible?